### PR TITLE
Enable masked systemd services in the frontend (HMS-3661)

### PR DESCRIPTION
**Pull request by kingsleyzissou**
Add masked services as a customization option, since disabling systemd units when the package isn't installed breaks the image build.

This depends on:
https://github.com/osbuild/image-builder/pull/1062 (manually added the API endpoint for the time being)

### CODE REVIEW

**Comment by lucasgarfield** [comment]
Awesome! Would we also want to include updates to the V2 wizard in this PR?

**Comment by lucasgarfield** [comment]
Actually, I just looked at where we are in the V2 Wizard with respect to the OpenSCAP test, and let's hold off on that for now... we can just prioritize getting this into V1.

**Comment by lavocatt** [comment]
Thanks @kingsleyzissou for making this PR! I've noticed that the checks are failing on the `api` checks, did you run the command `npm run api` on your local copy to regenerate the `api` files?

**Comment by kingsleyzissou** [comment]
> Thanks @kingsleyzissou for making this PR! I've noticed that the checks are failing on the `api` checks, did you run the command `npm run api` on your local copy to regenerate the `api` files?

Yeah that's the command I used to generate the api files.

**Comment by lavocatt** [comment]
> Yeah that's the command I used to generate the api files.

I've ran the command on your PR on my machine and I've got also some differences. That's strange, maybe something to do with the setup of the machine itself?

**Comment by kingsleyzissou** [comment]
I noticed I had issues with the version of node I was using, so I upgraded and then the output was different. I also may have done something silly. Will try again

**Comment by kingsleyzissou** [comment]
> I've ran the command on your PR on my machine and I've got also some differences. That's strange, maybe something to do with the setup of the machine itself?

It was my formatter. It edited on save :/

**Comment by lavocatt** [comment]
> > I've ran the command on your PR on my machine and I've got also some differences. That's strange, maybe something to do with the setup of the machine itself?
> 
> It was my formatter. It edited on save :/

It happens :smile: Did you push the new version without the editor getting in the way? Just wondering because the last version of the PR still has some quotes issues.

Maybe you need to specify to your local linter to ignore the autogenerated files

**Comment by kingsleyzissou** [comment]
> > > I've ran the command on your PR on my machine and I've got also some differences. That's strange, maybe something to do with the setup of the machine itself?
> 
> > 
> 
> > It was my formatter. It edited on save :/
> 
> 
> 
> It happens :smile: Did you push the new version without the editor getting in the way? Just wondering because the last version of the PR still has some quotes issues.
> 
> 
> 
> Maybe you need to specify to your local linter to ignore the autogenerated files

So weird. I didn't even open the file this time 🫠 I will try see what's going on

File: src/Components/CreateImageWizard/CreateImageWizard.js
Comment (regexowl) [inline src/Components/CreateImageWizard/CreateImageWizard.js:119]: Should this be `.masked` instead of `.disabled`?

**Comment by regexowl** [comment]
Went through the changes and the code looks good to me, thank you! :sparkles: Ready to approve this one after the small fix and rebase.

**Review by regexowl — APPROVED** [review]
LGTM, thank you!

**Pull request merged**
